test/prerna/testing/ApiSemossTestEngineUtils.java

```diff
@@ -45,7 +45,6 @@ public class ApiSemossTestEngineUtils {
 	private static Path ENGINES_CONFIG_FILE = Paths.get(ApiTestsSemossConstants.TEST_CONFIG_DIRECTORY.toString(), "engines.txt");
 	private static List<String> CORE_DBS = null;
 	
-	private static List<String> CURRENT_NAMES = new ArrayList<>();

```

Comment:
no longer checking current names, going to clean non core dbs after each test, unless if test class checks variable to not run the db cleaning method.